### ISSUE TITLE
mergeScripts should insert semicolons between <script> tags

### ISSUE THREAD

**Issue by jordanbtucker** [posthtml/htmlnano#55]
With the following HTML and default options:

```html
<script>document.write('Hello, ')</script><script>document.write('World!')</script>
```

It should produce:

```html
<script>document.write('Hello, ');document.write('World!');</script>
```

But instead you get the following error:
```bash
Error: SyntaxError: Unexpected token: name (document)
```

So it appears that it is concatenating the two <script> tags as this (without a semicolon in between):

```html
<script>document.write('Hello, ')document.write('World!')</script>
```

If you put the semicolon there yourself, `mergeScripts` does something even stranger. It replaces it with a comma:

```html
<script>document.write('Hello, ');</script><script>document.write('World!')</script>
```

becomes

```html
<script>document.write('Hello, '),document.write('World!');</script>
```

The comma is valid, but it does seem an odd choice over a semicolon.

**Comment by maltsev** [comment]
Thanks for the bug report! I fixed it. I'll release a new version today.

**Comment by maltsev** [comment]
> If you put the semicolon there yourself, mergeScripts does something even stranger. It replaces it with a comma:

Actually, that is a result of `minifyJs` optimization. If you disable it `;` shouldn't be replaced with `,` in that case.

**Comment by maltsev** [comment]
Released: [0.1.10](https://www.npmjs.com/package/htmlnano/v/0.1.10).
